### ISSUE TITLE
Find in Reference disabled for various keywords (draw, for, if, catch, while)

### ISSUE THREAD

**Issue by KevinWorkman** [processing/processing#5562]
Possibly related to #5555

From this forum post: https://discourse.processing.org/t/reference-and-help-tabs-greyed-out/1340

I have this code into my Processing editor:

```java
void draw() {
  for (int i = 0; i < 10; i++) {
    if (i < 5) {
      println(i);
    } else {
      println(i);
    }

    try {
      println(i);
    }
    catch(Exception e) {
      println(i);
    }
    
    do{
     println(i); 
    } while(false);
    
    while(true){
     println(i); 
    }
  }
}
```

If I right-click on various keywords, I'll get a menu, and one of the items on that menu is **Find in Reference**, which takes me to a local copy of the reference page for that keyword.

This works fine for `int`, `else`, `println()`, `try`, `true`, and `false`.

However, for many other keywords I see that option grayed out in the menu. This is the case for `draw()`, `for`, `if`, `catch`, and `while`.

I'm using Java mode, and all of those keywords are in my `keywords.txt` file, and they all have a reference page.

**Comment by benfry** [comment]
Huh, not sure what's happening there. Thanks for the report.

**Comment by svr8** [comment]
Hey! Found the error in [keywords.txt](https://github.com/processing/processing/blob/master/java/keywords.txt) and references.zip. Please review this PR: https://github.com/processing/processing/pull/5642

**Comment by GRsni** [comment]
This issue should probably gets closed too, seeing #5555 was closed already.

**Comment by benfry** [comment]
No, #5555 was closed as a duplicate because this has a better description of the problem.

**Comment by benfry** [comment]
We're on the trail of the problem, but @REAS is digging into it a bit more since it involves rebuilding the reference and `keywords_base.txt`.
